fictadvisor-api/src/v2/api/controllers/PollController.ts

```diff
@@ -153,7 +198,27 @@ export class PollController {
   @ApiBadRequestResponse({
     description: `\n
     InvalidBodyException:
-      Type must be enum
+      Order must not be greater than 50
+      Order must not be less than 1
+      Order can not be empty
+      Name must be shorter than or equal to 50 characters
+      Name must be longer than or equal to 5 characters
+      Name can not be empty
+      Text must be shorter than or equal to 250 characters
+      Text must be longer than or equal to 5 characters
+      Text can not be empty
+      Category must be shorter than or equal to 50 characters
+      Category must be longer than or equal to 5 characters
+      Category can not be empty
+      Description must be shorter than or equal to 2000 characters
+      Criteria must be shorter than or equal to 2000 characters
+      Type is not an enum
+      Requirement parameter is not a boolean
+      Role must be one of the following values: LECTURER, LABORANT, PRACTICIAN, EXAMINER, OTHER
+      Visibility parameter cannot be empty
+      Visibility parameter is not a boolean
+      Requirement parameter cannot be empty
+      Requirement parameter is not a boolean
 
     InvalidEntityIdException:
       Question with such id is not found`,
```

Comment:
Modify this `@Patch('/questions/:questionId')` endpoint. You should not add `roles` to the body. Check [your issue](https://github.com/fictadvisor/fictadvisor/issues/89) once again.